bitgouel-api/src/main/kotlin/bitgouel/team/msg/common/DataInitializer.kt

```diff
@@ -0,0 +1,197 @@
+package bitgouel.team.msg.common
+
+import org.springframework.boot.context.event.ApplicationReadyEvent
+import org.springframework.context.event.EventListener
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+import team.msg.common.logger.LoggerDelegator
+import team.msg.domain.club.model.Club
+import team.msg.domain.club.repository.ClubRepository
+import team.msg.domain.school.enums.HighSchool
+import team.msg.domain.school.model.School
+import team.msg.domain.school.repository.SchoolRepository
+import java.util.*
+
+@Component
+class DataInitializer(
+    private val schoolRepository: SchoolRepository,
+    private val clubRepository: ClubRepository
+) {
+
+    private val log by LoggerDelegator()
+
+    companion object {
+        val schoolList = listOf(
+            School(1, HighSchool.GWANGJU_TECHNICAL_HIGH_SCHOOL),
+            School(2, HighSchool.KUMPA_TECHNICAL_HIGH_SCHOOL),
+            School(3, HighSchool.JEONNAM_TECHNICAL_HIGH_SCHOOL),
+            School(4, HighSchool.GWANGJU_GIRLS_COMMERCIAL_HIGH_SCHOOL),
+            School(5, HighSchool.JEONNAM_GIRLS_COMMERCIAL_HIGH_SCHOOL),
+            School(6, HighSchool.GWANGJU_NATURAL_SCIENCE_HIGH_SCHOOL),
+            School(7, HighSchool.GWANGJU_ELECTRONIC_TECHNICAL_HIGH_SCHOOL),
+            School(8, HighSchool.DONGIL_HIGH_SCHOOL_OF_FUTURE_SCIENCE_HIGH_SCHOOL),
```

Comment:
HIGH_SCHOOL이 두 번 들어가 있는 게 어색하네요!! `DONGIL_HIGH_SCHOOL_OF_FUTURE_SCIENCE`가 공식 학교 영명이니 바꾸는 게 나아보입니다 🙄
```suggestion
            School(8, HighSchool.DONGIL_HIGH_SCHOOL_OF_FUTURE_SCIENCE),
```